composer.json

```
@@ -1,17 +1,17 @@
 {
-  "name": "katzwebservices/EDD-Gravity-Forms-Checkout",
+  "name": "easydigitaldownloads/EDD-Gravity-Forms-Checkout",
```

```suggestion
  "name": "easydigitaldownloads/edd-gravity-forms-checkout",
```

To match the spacing in our repo name.